apps/server/.env.example

```
@@ -0,0 +1,47 @@
+# --- SSRF egress guard: allow loopback for local dev ----------------------
+# The egress guard is ALWAYS ON and refuses loopback/private targets by default.
+# For local dev the hub must reach the client on loopback:
+#   FETCH allowlist    -> topic re-fetch on ping (the client's feed)
+#   CALLBACK allowlist -> WebSub verification GET + delivery, and the rssCloud
+#                         challenge GET / notify sent to the callback
+# Covers 127.x (IPv4) and ::1 (IPv6) since `localhost` may resolve to either.
+#
+# >>> PRODUCTION: delete both lines (or scope them to your real private feed /
+# >>> subscriber ranges). Never ship loopback exemptions to a public hub.
+WEBSUB_FETCH_ALLOW_CIDRS=127.0.0.0/8,::1/128
+WEBSUB_CALLBACK_ALLOW_CIDRS=127.0.0.0/8,::1/128
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WEBSUB_CALLBACK_ALLOW_CIDRS key should go before the WEBSUB_FETCH_ALLOW_CIDRS key

(UnorderedKey)